### PR TITLE
Fix new themes ready for packagist

### PR DESCRIPTION
**Pull request by RCheesley**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | 
| Deprecations?                          | 
| BC breaks? (use the c.x branch)        | 
| Automated tests included?              | 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

In https://github.com/mautic/mautic/pull/14155 we haven't been correctly set up each theme with a separate repo and Packagist package or configured the git split settings, so they are not going to be picked up by Composer installations. 

Each one should have a Packagist package, connected to the repository, and that should be pulled in [here](https://github.com/mautic/mautic/blob/5.x/.github/subtree-splitter-config.json) (which controls the git split, pushing changes / tags etc to the themes in core into the relevant repositories) and marked in the Composer file here: https://github.com/mautic/mautic/blob/5.x/composer.json#L53 this way they can be selectively added or removed from the build and in the future managed via the marketplace. 

This PR:

* Adds the git split configuration to connect the theme folder in mautic/mautic with the individual theme repos on GitHub
* Adds a GitHub Workflow for each sub-repo which we use to auto-close any PRs accidentally made in the sub-repos

Once this is merged, the sub-repos should be populated by the git split GitHub Action and then we can add the packages to composer.json (because we can't add an empty repo on Packagist - it requires the composer.json file).

### 📋 Steps to test this PR:

Please do code review as it won't be fully working until we get the Composer part sorted out.

The theme repos can be found using this query: https://github.com/orgs/mautic/repositories?type=all&q=theme.

### CODE REVIEW

File: .github/subtree-splitter-config.json
Comment (matbcvo) [inline .github/subtree-splitter-config.json]: Rename from `theme-education` to `theme-educate`

https://github.com/mautic/mautic/blob/b0d683f77b9d968bba72c343db642e703ac6e8aa/themes/_educate/composer.json#L2

File: .github/subtree-splitter-config.json
Comment (matbcvo) [inline .github/subtree-splitter-config.json]: Rename from `themes/education` to `themes/educate`

File: .github/subtree-splitter-config.json
Comment (matbcvo) [inline .github/subtree-splitter-config.json]: Rename from `theme-education.git` to `theme-educate.git`
Also rename repository from `mautic/theme-education` to `mautic/theme-educate`

https://github.com/mautic/mautic/blob/b0d683f77b9d968bba72c343db642e703ac6e8aa/themes/_educate/composer.json#L2

**Review by matbcvo — CHANGES_REQUESTED** [review]

File: .github/subtree-splitter-config.json
Comment (escopecz) [reply .github/subtree-splitter-config.json]: Yep, once this is done this should be ready to merge.

**Comment by RCheesley** [comment]
Thanks for the review folks, grateful for another pair of eyes! 👀 

I've addressed that mistake and also changed the name of the [theme repository](https://github.com/mautic/theme-educate).

**Review by matbcvo — APPROVED** [review]

**Review by escopecz — APPROVED** [review]
Thanks! 👍

**Pull request merged**
